demos/python_demos/multi_camera_multi_person_tracking/README.md

```diff
@@ -32,16 +32,22 @@ and then for each detected object it extracts embeddings using re-identification
 To install required dependencies run
 
 ```bash
-pip3 install -r requirements.txt
+cat requirements.txt | xargs -n 1 -L 1 pip3 install
```

Comment:
Why is this necessary?

Reply:
Some of modules in the list depend on `setuptools` or `Cython` and the first command `pip3 install -r requirements.txt` is crashed when you have clean system. You can check it with clean `virtualenv` and key `--no-cahce` to avoid using cached modules. So, the new command works without any troubles.

Reply:
I have tested this, and it seems that the problematic package is `pycocotools`. And I know that `pycocotools` is also a problem because it fails to install on WIndows. Is there any way you can make due without it?

Reply:
Yes, you are right the problem is in `pycocotools` but it is a necessary module. At least via our command `pycocotools` is installed properly in Linux. I would like to remain this command as is and we can not remove `pycocotools`. So, with Windows, as I understand, it is a global problem and if we need this module any way.

Reply:
> but it is a necessary module.

Is it, though?

I have tried to figure out how it is used, and it looks to me like the code where it's used is unreachable. The only place where it's used is in `utils.segm_postrocess.segm_postprocess` when `encode` is true, but nothing seems to actually call it with `encode=True`.

Reply:
Yes, you are right. Unused code removed, `pycocotools removed` too. Command to install python modules restored.

Reply:
Alright, much better.